packaging/linux/openra-mimeinfo.xml

```diff
@@ -8,4 +8,11 @@
     <glob weight="60" pattern="*.orarep"/>
   </mime-type>
 
+  <mime-type type="x-scheme-handler/openra">
+    <icon name="openra" />
+    <generic-icon name="applications-games"/>
+    <comment>OpenRA ServerAccess Scheme</comment>
```

Comment:
https://www.google.co.nz/search?q=%22ServerAccess+scheme%22 appears to be terminology specific to Hedgewars.